etl/xml_importer/xpaths.py

```diff
@@ -29,18 +29,15 @@
     # LOCATION
     "Artwork_Location_Path": 'lido:descriptiveMetadata/lido:objectIdentificationWrap/lido:repositoryWrap/lido:repositorySet',
     "Location_Label_Path": 'lido:repositoryName/lido:legalBodyName/lido:appellationValue',
-
     "Location_PlaceID_Path": 'lido:repositoryLocation/lido:placeID[@lido:source]',
-    "Location_Name_Path": 'lido:repositoryLocation/lido:namePlaceSet/lido:appellationValue',    #Name and alte_Name have the same path
-    "Location_Altname_Path": 'lido:repositoryLocation/lido:namePlaceSet/lido:appellationValue',
-
-    "Location_PlaceLabel_Path": 'lido:repositoryLocation/lido:namePlaceSet/lido:appellationValue',##Same path as Name and Location_Name_Path
+    "Location_inventoryNumber": 'lido:workId[@lido:type="Inventarnummer"]',
```

Comment:
The name of the path should be `Location_InventoryNumber_Path`
This Path seems to be wrong. The correct path is:
```suggestion
    "Location_InventoryNumber_Path": 'lido:workID[@lido:type="Inventarnummer"]',
```